content/english/supply chain analytics/1_Introduction.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Introduction to Supply Chain"
```

Comment:
I would consider having all this in the _index.md file, so that students do not need to click an extra page to jump into the content